### PR TITLE
Template: add option to exclude license from pipeline template

### PR DESCRIPTION
**Pull request by mirpedrol**
Part of https://github.com/nf-core/tools/issues/2340 and https://github.com/nf-core/tools/issues/2342

### CODE REVIEW

File: nf_core/pipelines/create/templatefeatures.yml [hunk 1]
```diff
@@ -242,3 +242,18 @@ changelog:
       - "CHANGELOG.md"
   nfcore_pipelines: False
   custom_pipelines: True
+license:
+  skippable_paths:
+    - "LICENSE"
+  short_description: "Add a license"
```
Comment (mashehu) [inline nf_core/pipelines/create/templatefeatures.yml:248]:
```suggestion
  short_description: "Add a license File"
```

File: nf_core/pipelines/create/templatefeatures.yml [hunk 2]
```diff
@@ -242,3 +242,18 @@ changelog:
...
+  description: "Add the MIT license file."
```
Comment (mashehu) [inline nf_core/pipelines/create/templatefeatures.yml:249]: How about we add also a link to the requirement docs here?

**Review by mashehu — APPROVED** [review]

File: nf_core/pipelines/create/templatefeatures.yml (hunk 2 above)
Comment (mirpedrol) [reply nf_core/pipelines/create/templatefeatures.yml:249]: Which docs? 🤔

Comment (mashehu) [reply nf_core/pipelines/create/templatefeatures.yml:249]: https://nf-co.re/docs/guidelines/pipelines/requirements/mit_license

Comment (mirpedrol) [reply nf_core/pipelines/create/templatefeatures.yml:249]: ah ok! But this is only for nf-core pipelines, and the option to not add a license is for non-nf-core ones.

**Pull request merged**